public/friendlycode/js/require-config.js

```diff
@@ -35,7 +35,7 @@ var require = {
   packages: ['slowparse-errors'],
   paths: {
     // Vendor paths
-    "jquery.min": "../vendor/jquery.min",
+    "jquery.min": "https://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min",
     "jquery-ui": "../vendor/jquery-ui-1.11.3.custom/jquery-ui.min",
```

Comment:
right now we load our own custom build of jquery-ui that's a lot smaller than the entire library